spec/supabase_js_v2.yml

```diff
@@ -345,6 +345,26 @@ pages:
           ```js
             const { data, error } = supabase.auth.setSession(refresh_token)
           ```
+  auth.refreshSession():
+    title: 'refreshSession()'
+    $ref: '@supabase/gotrue-js.GoTrueClient.refreshSession'
+    notes: |
+      - This method will refresh the session whether the current one is expired or not.
+      - For server-side use, pass in the current session which includes a refresh token; as `getSession()` likely won't be able to retrieve it.
```

Comment:
This should not be recommended because it's unsafe. If both the client and server are using the same refresh token, the client won't be aware that the refresh token has been used on the server-side. When the client tries to use the refresh token, it will lead to the reuse detection kicking in and all descending refresh tokens will be revoked.